PAT docs for Konnect (#4499)
Kong/docs.konghq.com: app/konnect/runtime-manager/runtime-groups/declarative-config.md

* decK v1.12.0 or later [installed](/deck/latest/installation/).
* Optional: To test your configuration, [set up a simple runtime](/konnect/getting-started/configure-runtime).

## Generate a Personal Access Token

> To test your connection, it is recommended that you use a personal access token (PAT).

Reviewer: ```suggestion
To test your connection, we recommend that you use a personal access token (PAT).
```
Going by what google does, and also just a personal preference. This is definitely a point of discussion for us though - what should the phrasing be, so that it's consistent across our docs? I stuck it into our S&G discussion list for now.
